metadata-service/war/src/main/resources/boot/retention.yaml

```diff
@@ -0,0 +1,14 @@
+- entity: "*"
+  aspect: "*"
+  retention:
```

Comment:
minor nit: do we need the extra "retention" field?

could it be 

- entity
   aspect
   retentionPollicies

Reply:
yeah. that's ideal, but I wanted to simply convert the section into DataHubRetentionInfo object without adding more logic there, which requires this field